File: examples/Material/test_mob.cc
#include <iostream>
#include "PMI_benchmark_mob.h"


int main(int argc, char ** args)
{
  PMI_Benckmark_Mob * mob = new PMI_Benckmark_Mob("Si", "Lucent");
  mob->set_doping(1e18, 0.0);
  std::cout<<mob->mob_electron(1e18, 0, 0, 0, 300)<<std::endl;

  delete mob;
}

File: examples/Material/semiconductor_benchmark/sip/test_mob.py
import os
from PMI_Benchmark import *


mob = PMI_Benchmark_Mob("/home/gdiso/develop/genius/lib/", "Si", "Lucent")
mob.calibrate_real_parameter("NRFP.UM", 3e17)

mob.set_doping(1e18, 0.0)
print mob.mob_electron(1e18, 0, 0, 0, 300)

